packages/ajna-contracts/scripts/common/deployment.utils.ts

```diff
@@ -180,14 +180,17 @@ export async function deployPoolFactory(
 export async function deployPool(
   erc20PoolFactory: ERC20PoolFactory,
   collateral: string,
-  quote: string
+  quote: string,
+  deployPools = true
 ): Promise<ERC20Pool> {
   const hash = await erc20PoolFactory.ERC20_NON_SUBSET_HASH();
   let poolAddress = await erc20PoolFactory.deployedPools(hash, collateral, quote);
-  if (poolAddress === hre.ethers.constants.AddressZero) {
-    await erc20PoolFactory.deployPool(collateral, quote, "50000000000000000", {
+  if (poolAddress === hre.ethers.constants.AddressZero && deployPools) {
+    console.log("Deploying pool");
+    const tx = await erc20PoolFactory.deployPool(collateral, quote, "25000000000000000", {
```

Comment:
Are we setting the same rate for every pool?

Reply:
yes, we already did :) pools are on mainnet